ViewGenerator/build/ViewGenerator.targets

```diff
@@ -29,7 +33,7 @@
     <Delete Files="@(JSFilesToDelete)"/>
   </Target>
   
-  <Target Name="IncludeGeneratedResources" DependsOnTargets="FindConfigFiles;CompileTypeScriptWithTSConfig">
+  <Target Name="IncludeGeneratedResources" DependsOnTargets="FindConfigFiles;CompileTypeScriptWithTSConfig;GenerateBundle">
     <ItemGroup> 
       <EmbeddedResource Include="**\*.js" />
```

Comment:
Will it include the non bundle files?

Reply:
Yes, but I will remove the targets related with typescript compilation, as they're not necessary anymore. Therefore, the only .js files included will be the bundles

Reply:
Ok.. in the end make sure you check the assembly produced to check that the only js files there are the bundles

Reply:
Checked